README.md

```
@@ -30,17 +30,46 @@
-- Remove all the spaces such that the whole _json_ that you copied is in a single line
-- Replace **\n** with **\\\\n** in your copied json
-- Replace **"** with **\\"** in your copied json
-### *Firestore setup before running the server locally*
-- Create an application on [FireStore](https://firebase.google.com/docs/firestore) and [generate a service file](https://cloud.google.com/iam/docs/creating-managing-service-account-keys).
-- Add the service file credentials in the sample-env.json file as a string.
-- Rename `sample-env.json` to `env.json`
-
-### *Executing the script to run the server locally*
+# Setting Up Environment for Local Server
+
+## 1. Firestore Setup
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)